test/test_nn.py

```diff
@@ -1800,6 +1800,10 @@ def add_test(test):
         test_cuda=False
     ),
     dict(
+        constructor=lambda: nn.CosineSimilarity(),
+        input_size=(2,5,7),
```

Comment:
nit: could you add spaces between the elements? `(2, 5, 7)` (for the lint checker)